packages/livekit-server-sdk/src/ConnectorClient.ts

```diff
@@ -0,0 +1,283 @@
+  whatsappCallId: string;
+  /** Optional - An arbitrary string you can pass in that is useful for tracking and logging purposes */
+  whatsappBizOpaqueCallbackData?: string;
+  /** Required - The call accept webhook comes with SDP from Meta */
+  sdp: SessionDescription;
+  /** Optional - What LiveKit room should this participant be connected to */
+  roomName?: string;
+  /** Optional - Agents to dispatch the call to */
+  agents?: RoomAgentDispatch[];
+  /** Optional - Identity of the participant in LiveKit room */
+  participantIdentity?: string;
+  /** Optional - Name of the participant in LiveKit room */
+  participantName?: string;
+  /** Optional - User-defined metadata. Will be attached to a created Participant in the room. */
+  participantMetadata?: string;
+  /** Optional - User-defined attributes. Will be attached to a created Participant in the room. */
+  participantAttributes?: { [key: string]: string };
+  /** Optional - Country where the call terminates as ISO 3166-1 alpha-2 */
+  destinationCountry?: string;
+}
+
+// Twilio types
+export interface ConnectTwilioCallOptions {
+  /** The direction of the call */
+  twilioCallDirection: ConnectTwilioCallRequest_TwilioCallDirection;
+  /** What LiveKit room should this call be connected to */
+  roomName: string;
+  /** Optional agents to dispatch the call to */
+  agents?: RoomAgentDispatch[];
+  /** Optional identity of the participant in LiveKit room */
+  participantIdentity?: string;
+  /** Optional name of the participant in LiveKit room */
+  participantName?: string;
+  /** Optional user-defined metadata. Will be attached to a created Participant in the room. */
+  participantMetadata?: string;
+  /** Optional user-defined attributes. Will be attached to a created Participant in the room. */
+  participantAttributes?: { [key: string]: string };
+  /** Country where the call terminates as ISO 3166-1 alpha-2 */
+  destinationCountry?: string;
+}
+
+/**
+ * Client to access Connector APIs for WhatsApp and Twilio integrations
+ */
+export class ConnectorClient extends ServiceBase {
+  private readonly rpc: Rpc;
+
+  /**
+   * @param host - hostname including protocol. i.e. 'https://<project>.livekit.cloud'
+   * @param apiKey - API Key, can be set in env var LIVEKIT_API_KEY
+   * @param secret - API Secret, can be set in env var LIVEKIT_API_SECRET
+   * @param options - client options
+   */
+  constructor(host: string, apiKey?: string, secret?: string, options?: ClientOptions) {
+    super(apiKey, secret);
+    const rpcOptions = options?.requestTimeout
+      ? { requestTimeout: options.requestTimeout }
+      : undefined;
+    this.rpc = new TwirpRpc(host, livekitPackage, rpcOptions);
+  }
+
+  /**
+   * Initiate an outbound WhatsApp call
+   *
+   * @param options - WhatsApp call options
+   * @returns Promise containing the WhatsApp call ID and room name
+   */
+  async dialWhatsAppCall(options: DialWhatsAppCallOptions): Promise<DialWhatsAppCallResponse> {
```

Comment:
`DialWhatsAppCallResponse` and the other response types need to be re-exported as well